astroid/brain/brain_attrs.py

```diff
@@ -0,0 +1,38 @@
+"""Astroid hook for the attrs library
+
+Without this hook pylint reports unsupported-assignment-operation
+for atrrs classes
+"""
+import astroid
+from astroid import MANAGER
+
+
+def register(linter):
```

Comment:
No need to have this, since it is not a pylint plugin. We already do the registration through the manager